docs/tutorial/workers.md

```diff
@@ -0,0 +1,165 @@
+
+1. [A flow](/tutorial/flows/)
+2. A work pool
+3. A worker
+
+Now it’s time to put it all together.
+
+In your terminal (not the terminal in which the worker is running), navigate to your `repo_info.py` file that we created in the first section; for best results, this file should be in its own _empty directory_. Now run the following command ***from the root of this directory*** to begin deploying your flow:
+
+<div class="terminal">
+```bash
+prefect deploy
+```
+</div>
+
+!!! tip "Specifying an entrypoint"
+    In non-interactive settings (like CI/CD), you can specify the entrypoint of your flow directly in the CLI. 
+    
+    For example, if `get_repo_info` is defined in `repo_info.py`, provide deployment details with flags `prefect deploy repo_info.py:get_repo_info -n my-deployment -p my-docker-pool`.
+
+When running `prefect deploy` interactively, the CLI will discover all flows in your working directory. Select the flow you want to deploy, and the deployment wizard will walk you through the rest of the deployment creation process:
+
+1. **Deployment name**: Choose a name, like `my-deployment`
+2. **Would you like to configure a schedule for this deployment? (y/n):** Type `n` for now, you can set up a schedule later
+3. **Which work pool would you like to deploy this flow to? (use arrow keys):** Select the work pool you just created, `my-docker-pool`
+4. **Would you like to build a custom Docker image for this deployment? (y/n):** Select `y` to have Prefect build an image for you
+5. **Repository name (e.g. your Docker Hub username):** For the purposes of the tutorial, you can input anything you'd like here
+6. **Image name (my-first-deployment):** Hit `Enter` to use the image name default
+7. **Image tag (latest):** Hit `Enter` to use the image tag default
+8. **Would you like to push this image to a remote registry? (y/n):** Select `n` for now; we can keep this image local
+
+Prefect will now build a custom Docker image containing your workflow code that the worker can use to dynamically spawn Docker containers whenever this workflow needs to run; try it out:
+
+<div class="terminal">
+```bash
+prefect deployment run 'get_repo_info/my-first-deployment'
+```
+</div>
+
+!!! danger "Common Pitfalls"
+    - When running `prefect deploy`, double check that you are at the **root of your repo**, otherwise the worker may attempt to use an incorrect flow entrypoint during remote execution!
+    - Ensure that you have pushed any changes to your flow script to your GitHub repo - at any given time, your worker will pull the code that exists there!
+
+As you continue to use Prefect, you'll likely author many different flows and deployments of them. Check out the next section to learn about defining deployments in a `prefect.yaml` file.
+
+!!! tip "Did you know?"
+    A Prefect flow can have more than one deployment. This can be useful if you want your flow to run in different execution environments or have multiple different schedules.
```

Comment:
```suggestion
    A Prefect flow can have more than one deployment. This can be useful if you want your flow to run in different execution environments or have multiple schedules.
```